<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableManager</class>
 <widget class="QWidget" name="TableManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1574</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59, 59, 59);
border-color: rgb(44, 44, 44);</string>
  </property>
  <widget class="QLabel" name="Welcome">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; font-style:italic; color:#e7af65;&quot;&gt;WELCOME TO &lt;/span&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; font-style:italic; color:#e7af65;&quot;&gt;NICK'S BILLIARDS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Table" name="table1" native="true">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>138</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="toolTipDuration">
    <number>12</number>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table2" native="true">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>215</y>
     <width>115</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table3" native="true">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>293</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table4" native="true">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>370</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table5" native="true">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>138</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table6" native="true">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>215</y>
     <width>115</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table7" native="true">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>293</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table8" native="true">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>370</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table9" native="true">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>138</y>
     <width>114</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table10" native="true">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>215</y>
     <width>114</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table11" native="true">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>293</y>
     <width>114</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table12" native="true">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>370</y>
     <width>114</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table13" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>138</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table14" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>215</y>
     <width>115</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table15" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>293</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table16" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table17" native="true">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>138</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table18" native="true">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>215</y>
     <width>115</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table19" native="true">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>293</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table20" native="true">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>370</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table21" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>138</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table22" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>215</y>
     <width>115</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table23" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>293</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Table" name="table24" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>370</y>
     <width>115</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="Control" name="control" native="true">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>140</y>
     <width>321</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:35px;
background:rgb(223,167,40)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>461</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(45,120,92)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>460</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(224, 59, 62);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>  9 footer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>461</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>   7 footer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>460</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string> Snooker</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>560</y>
     <width>741</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(68, 68, 68);</string>
   </property>
   <property name="placeholderText">
    <string>Let your team members know of the goingons here </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>540</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Memo/Specials</string>
   </property>
  </widget>
  <widget class="TableTransfer" name="transfer" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>570</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>560</y>
     <width>351</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dailySummray_pushButton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>10</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/info.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoRepeatDelay">
    <number>1</number>
   </property>
   <property name="autoRepeatInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>130</y>
     <width>341</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(68, 68, 68);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>label_8</zorder>
  <zorder>Welcome</zorder>
  <zorder>table1</zorder>
  <zorder>table2</zorder>
  <zorder>table3</zorder>
  <zorder>table4</zorder>
  <zorder>table5</zorder>
  <zorder>table6</zorder>
  <zorder>table7</zorder>
  <zorder>table8</zorder>
  <zorder>table9</zorder>
  <zorder>table10</zorder>
  <zorder>table11</zorder>
  <zorder>table12</zorder>
  <zorder>table13</zorder>
  <zorder>table14</zorder>
  <zorder>table15</zorder>
  <zorder>table16</zorder>
  <zorder>table17</zorder>
  <zorder>table18</zorder>
  <zorder>table19</zorder>
  <zorder>table20</zorder>
  <zorder>table21</zorder>
  <zorder>table22</zorder>
  <zorder>table23</zorder>
  <zorder>table24</zorder>
  <zorder>control</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>plainTextEdit</zorder>
  <zorder>label_7</zorder>
  <zorder>dailySummray_pushButton</zorder>
  <zorder>transfer</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Table</class>
   <extends>QWidget</extends>
   <header>table.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableTransfer</class>
   <extends>QWidget</extends>
   <header>tabletransfer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Control</class>
   <extends>QWidget</extends>
   <header location="global">control.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
